Source/Core/Core/HW/WiimoteEmu/Attachment/Nunchuk.cpp

```diff
@@ -125,9 +154,9 @@ void Nunchuk::GetState(u8* const data)
   s16 accel_y = (s16)(4 * (accel.y * ACCEL_RANGE + ACCEL_ZERO_G));
   s16 accel_z = (s16)(4 * (accel.z * ACCEL_RANGE + ACCEL_ZERO_G));
 
-  accel_x = MathUtil::Clamp<s16>(accel_x, 0, 1024);
-  accel_y = MathUtil::Clamp<s16>(accel_y, 0, 1024);
-  accel_z = MathUtil::Clamp<s16>(accel_z, 0, 1024);
+  accel_x = MathUtil::Clamp<s16>(accel_x, 0, 0x3ff);
```

Comment:
Just curious, how did you determine this was off by one?

Reply:
We're clamping to the range of a 10-bit unsigned value. i.e. 2^10-1.